src/IAM/UI/REST/Request/LogoutRequest.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Twitter\IAM\UI\REST\Request;
+
+final class LogoutRequest
+{
+    public function __construct(
+        public readonly string $refreshToken,
```

Comment:
Encapsulate properties inside the request and add getters to access them.

```suggestion
        private readonly string $refreshToken,
```